src/lib/ollama.ts

```diff
@@ -1,25 +1,18 @@
 import { get } from 'svelte/store';
 import { settingsStore } from '$lib/store';
+import type { ChatRequest, ListResponse } from 'ollama/browser';
 
-export type OllamaModel = {
-	name: string;
-	model: string;
-	modified_at: string;
-	size: number;
-	digest: string;
-	details: {
-		parent_model: string;
-		format: string;
-		family: string;
-		families: string[] | null;
-		parameter_size: string;
-		quantization_level: string;
-	};
-};
+export async function ollamaChat(payload: ChatRequest, abortSignal: AbortSignal) {
+	const settings = get(settingsStore);
+	if (!settings) throw new Error('No Ollama server specified');
 
-export type OllamaTagResponse = {
-	models: OllamaModel[];
-};
```

Comment:
Minor cleanup, instead of having our own custom types we can simply use the types as defined by `ollama-js`. One less thing to maintain.